identity/app/views/profile/accountDetailsForm.scala.html

```diff
@@ -150,6 +150,9 @@ <h2 class="form__heading">Personal information</h2>
     <fieldset class="fieldset">
         <div class="fieldset__heading">
             <h2 class="form__heading">Correspondence address</h2>
+            <div class="form-field__note">
+                If you have a newspaper or Guardian Weekly subscription and would like to update your delivery address, please <a href="https://www.theguardian.com/help/2017/dec/11/help-with-updating-your-contact-or-delivery-details" rel="noopener" target="_blank"> click here to contact our Customer Service team.</a>
```

Comment:
I'd remove the space between `blank">` and `click` so that the hyperlink underline doesn't trail